torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -292,27 +277,26 @@ bool Reducer::ddp_graph_static() {
 }
 
 void Reducer::initialize_local_used_map() {
-  const auto replica_count = replicas_.size();
-  const auto variable_count = replicas_[0].size();
+  const auto replica_count = 1;
+  size_t params_index = 0;
```

Comment:
why is params_index set as 0?

Reply:
Same reason that replica_count is set to 1, so that we can keep track of how the instances of indexing are related to each other. I can remove this